### PR TITLE
Update CHANGES for release 14.1.0

### PR DESCRIPTION
**Pull request by jeremy-lunarg**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -3,6 +3,35 @@
...
+* Add SPIRV-Tools-opt dependency if ENABLE_OPT
+* Support EXT_shader_quad_control
+* Add OpAssumeTrueKHR and OpExpectKHR
+* Support GL_EXT_maximal_reconvergence
+* Remove generation of deprecated Target.cmake files
+* Fix array size of gl_SampleMask and gl_SampleMaskIn
+* Support GL_ARB_texture_multisample_extension
+* Emit DebugTypePointer when non-semantic debug info is enabled
+* Add missing include file for MinGW build
+* Improve builtin versioning with modern C++ containers
```
Comment (arcady-lunarg) [inline CHANGES.md:33]: This is internal-only

File: CHANGES.md [hunk 2]
```diff
@@ -3,6 +3,35 @@
...
 
+## 14.1.0 2024-03-08
+* Add option to output absolute paths
+* Fix Visual Studio 2022 bitfield width warning (/w15249)
+* Support GL_EXT_control_flow_attributes2
+* Fix generation of debug info file and source strings
+* Support GL_ARB_shading_language_include
+* Fix HLSL built-in passthrough via inout
+* Enable -Wimplicit-fallthrough and fix warnings
+* Quote paths in runtest script
```
Comment (arcady-lunarg) [inline CHANGES.md:14]: I think this is kind of internal-only, or only relevant to people building and also testing glslang

File: CHANGES.md [hunk 3]
```diff
@@ -3,6 +3,35 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 
+## 14.1.0 2024-03-08
+* Add option to output absolute paths
+* Fix Visual Studio 2022 bitfield width warning (/w15249)
```
Comment (arcady-lunarg) [inline CHANGES.md:8]: This doesn't seem worth calling out.

File: CHANGES.md [hunk 4]
```diff
@@ -3,6 +3,35 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 
+## 14.1.0 2024-03-08
+* Add option to output absolute paths
```
Comment (arcady-lunarg) [inline CHANGES.md:7]: "Add a new --abosute-path command-line option to output absolute paths in error messages"

File: CHANGES.md [hunk 5]
```diff
@@ -3,6 +3,35 @@
...
+* Fix Visual Studio 2022 bitfield width warning (/w15249)
+* Support GL_EXT_control_flow_attributes2
+* Fix generation of debug info file and source strings
```
Comment (arcady-lunarg) [inline CHANGES.md:10]: This is just a change to the test harness, no need to include it.

**Review by arcady-lunarg — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
Comment (jeremy-lunarg) [reply CHANGES.md]: Removed.

File: CHANGES.md (hunk 2 above)
Comment (jeremy-lunarg) [reply CHANGES.md]: Removed.

File: CHANGES.md (hunk 3 above)
Comment (jeremy-lunarg) [reply CHANGES.md]: Removed.

File: CHANGES.md (hunk 4 above)
Comment (jeremy-lunarg) [reply CHANGES.md]: Updated.

File: CHANGES.md (hunk 5 above)
Comment (jeremy-lunarg) [reply CHANGES.md]: Removed.

**Review by arcady-lunarg — APPROVED** [review]
LGTM, thanks

**Pull request merged**